weave/__init__.py

```diff
@@ -37,6 +38,7 @@
     attributes,
     # Re-exported from flow module
     Object,
+    ImperativeEvaluationLogger,
```

Comment:
maybe we exclude this from this PR (effectively keeping it hidden). Then brining it to the top level is the "release moment".

Reply:
I also suspect Adrian/Scott/Shawn will have opinions about the name "ImparativeEvaluationLogger" lol

Reply:
we can just call it `EvaluationLogger` -- that is implicitly imperative I think